ST_species.c

```diff
@@ -429,7 +546,7 @@ void save_annual_species_relsize() {
             //printf("Globals.currYear = %d, sp=%d , Species[sp]->relsize=%.5f ,old value lastyear_relsize : %.5f \n", Globals.currYear, sp, Species[sp]->relsize, Species[sp]->lastyear_relsize);
             Species[sp]->lastyear_relsize = getSpeciesRelsize(sp);
             //Species[sp]->lastyear_relsize = 2;
-            //printf("Globals.currYear = %d, sp=%d new updated value lastyear_relsize : %.5f \n", Globals.currYear, sp, Species[sp]->lastyear_relsize);
+            //printf("Globals->currYear = %d, sp=%d new updated value lastyear_relsize : %.5f \n", Globals->currYear, sp, Species[sp]->lastyear_relsize);
```

Comment:
Can we get rid of this commented-out code?

Reply:
yes